content/10-minute-tutorial.md

```diff
@@ -1,3 +1,324 @@
 ---
 title: 10 Minute Tutorial
+polyglot: true
 ---
+{{% block "java" %}}
+
+To get started with Cucumber in Java, you will need the following:
+
+- [Java SE](http://www.oracle.com/technetwork/java/javase/downloads/index-jsp-138363.html) - Java 8 (Java 9 is not yet supported by Cucumber)
+
+- [Maven](https://maven.apache.org/index.html) - Version 3.3.1 or higher
+
+- An IDE editor, for example [IntelliJ IDEA](https://www.jetbrains.com/idea/?fromMenu#chooseYourEdition) (which will be used in this
+  introduction)
+
+- A Cucumber plugin for your IDE, for example [IntelliJ IDEA Cucumber for Java plugin](https://plugins.jetbrains.com/plugin/7212-cucumber-for-java) to go with IntelliJ IDEA
```

Comment:
Has this been fixed yet to work with cucumber-jvm 2?

Reply:
It hasn't. I see now.

Reply:
Should we mention anything about that?

Reply:
The IDE plugin is presumably optional? Maybe a link to working with  Cucumber outside of the IDE?

Reply:
Actually, not sure the plugin is optional with IntelliJ: https://www.jetbrains.com/help/idea/cucumber.html

Reply:
Eclipse one works great and is available in the marketplace, in case you want to mention it.

Reply:
@Flaburgan IntelliJ works fine otherwise, maybe when adding a tutorial with Eclipse. Thanks!